raster3d/r3.in.lidar/r3.in.lidar.html

```diff
@@ -207,8 +207,3 @@ <h2>REFERENCES</h2>
 <h2>AUTHOR</h2>
 
 Vaclav Petras, <a href="https://geospatial.ncsu.edu/geoforall/">NCSU GeoForAll Lab</a>
-
-<!--
-<p>
-<i>Last changed: $Date$</i>
--->
```

Comment:
`<h2>REFERENCES</h2>` section should be before `<h2>SEE ALSO</h2>`.

Reply:
thanks, fixed